templates/cdap-deployment.yaml

```diff
@@ -19,29 +19,29 @@ metadata:
   {{if .Base.Labels}}
   labels:
     {{range $k,$v := .Base.Labels}}
-    {{$k}}: {{$v}}
+    "{{$k}}": "{{$v}}"
```

Comment:
YAML keys are always string. Quote is not needed.

Reply:
I'm not sure that's the case, from the Yaml spec:
> The content of a mapping node is an unordered set of key: value node pairs, with the restriction that each of the keys is unique. YAML places no further restrictions on the nodes. In particular, keys may be arbitrary nodes, the same node may be used as the value of several key: value pairs, and a mapping could even contain itself as a key or a value (directly or indirectly).

[Source](https://yaml.org/spec/1.2/spec.html#id2764044)

That said, I'm happy to remove the quotes, I don't really think they're necessary for keys, it was just an overabundance of caution.